index.html

```diff
@@ -304,19 +306,24 @@ <h4><a href="./news/2019-07-02-Guelph-accessibility-conference.html">Floe at the
             </div>
 
             <footer class="floe-footer">
+                    <div class="floe-footer-icons">
+                            <div class="social-media-logos-container">
+                                <a href="https://twitter.com/FluidProject" target="_blank"><img class="social-media-logo" alt="Twitter" src="images/icon-twitter.svg" /></a>
+                                <a href="https://www.facebook.com/idrctoronto/" target="_blank"><img class="social-media-logo" alt="Facebook" src="images/icon-facebook.svg" /></a>
+                                <a href="https://github.com/fluid-project/floeproject.org" target="_blank"><img class="social-media-logo" alt="Github" src="images/icon-github.svg" /></a>
```

Comment:
that's the GitHub repo for the site itself, but not the projects being worked on related to the floe project. I understand that this is a bit confusing, but it's why we need to understand what to put here, if anything, first.

Reply:
Ok @jobara . I understand